vtm.js

```diff
@@ -359,6 +359,44 @@ const attributesAndSkillsMixin = {
   },
 }
 
+/**
+ * displays state of current resource assigment
+ * there are two input arrays
+ * allocatedResources array contains number of attributes assigned to that value
+ * resourceRestrictions array contains number of attributes allowed
+ * this component will display number of allocated points vs. allowed
+ * */
+app.component('restriction-state', {
+  props: ['allocatedResources', 'resourceRestrictions'],
+  computed: {    
+    lastInList() {
+      for (var i = 1; i < this.resourceRestrictions.length; i++) {  
+        if (this.allocatedResources[i] == 0 && this.resourceRestrictions[i] == 0) {
+          return i-1;
+        }
+      }
+    }    
+  },
+  template: `
+    <div class="resourceCount">
+      Divide: 
+      <span
+        v-for="(item, index) in resourceRestrictions"
+        v-show="item > 0 | allocatedResources[index] > 0"
+      >         
+        <span :class="{ red: allocatedResources[index] > item }">
+          {{ resourceRestrictions[index] - allocatedResources[index] }} x
+        </span>
+        <span v-if="index == 0" class="point">
+        </span>
+        <span v-for="number in index" class="point fill">
```

Comment:
body by mohli byt fill pokial niesu rozdelene a menit sa postupne na init ked budu, tzn ze to bude taky progress bar
ale neviem ci je to dobre, to si nejak nemozem odpustit nenapisat, ale asi mi to nevadi tak jak to je a nemylit uzivatela nejakou dalsou farbou... cisla by mohli byt dostatocna indikacia

Reply:
alebo este som si vsimol ze minuska robia to ze to mierne odskoci.
tak mozno by sa dalo len naznacit ze je underflow a ukazovat tam len ten pocet
alebo ma este napadlo ze ked tam je 0 tak je to OK a mohlo by to zmiznut a neotravovat, ale to by sme asi v tom hoveru potom nechceli